src/tests/ftest/util/server_utils.py

```diff
@@ -290,3 +273,35 @@ def kill_server(hosts):
     for host in hosts:
         subprocess.call("ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)),
                         shell=True)
+
+def clustershell_execute(cmd, hosts, timeout=60):
+    """
+    Python cluster shell execute function with error check.
+    Args:
+        cmd: Command in string to run on cluster
+        hosts: Cluster nodes
+        timeout: Command timeout
+    """
+    local_task = task_self()
+    local_task.run(cmd=cmd, nodes=hosts, timeout=timeout)
+    for rc_code, _keys in local_task.iter_retcodes():
+        if rc_code is not 0:
+            for output, _node in local_task.iter_buffers():
+                raise ServerFailed("Cluster Command Failed\n{} : RC={} :{}"
+                                   .format(_node, rc_code, output))
+
+def storage_prepare(hosts):
+    """
+    Prepare the Storage for server
+    """
+    cmd = ("sudo /usr/bin/daos_server storage prep-nvme "
```

Comment:
I guess this test is going to have to wait for Functional testing to be running from RPMs before we can land it, yes?

Reply:
Yes that is the one of the reason I was asking on Skype if we have any planned date from when we will use RPM in Ci

Reply:
No _planned_ date other than to say ASAP.

But there are tasks blocking that.  At least [DAOS-2260](https://jira.hpdd.intel.com/browse/DAOS-2260) or some child/related ticket to it which describes being able to run the Functional tests portably either in `./install/` or the system (`/usr/`) prefix.